client/store/globals.js

```diff
@@ -12,7 +11,21 @@ export const state = () => ({
   selectedCollection: null,
   selectedAuthor: null,
   isCasting: false, // Actively casting
-  isChromecastInitialized: false // Script loaded
+  isChromecastInitialized: false, // Script loaded
+  dateFormats: [
```

Comment:
I don't love my choice to put dateFormats in state since it's not really.
But I was struggling to access the list when I put it other places, and also I'm not sure where the right place to put it is.
I'm open to suggestions about this.

Reply:
I'm not sure we have a great place to put this yet. I think it's good here for now, no need to overthink it. We can consolidate data like this later.